draft-ietf-core-cf-reg-update.md

```diff
@@ -159,26 +162,30 @@ A new column with the title "Note" has been added to the registry, which contain
 
 ## "Full" Expert Review Checks {#full-checks}
 
-The registration procedure for the 0-255 range has been slightly modified -- from "Expert Review" to "Expert Review (Full)" -- to clearly distinguish it from the "lightweight" Expert Review that may apply to the 10000-64999 range.
-For the 0-255 range, in addition to the checks described in {{checks}}, the DE must also evaluate the requested codepoint concerning the limited availability of the 1-byte codepoint space.
+For the 0-255 range, the DE is instructed to perform a "Full Review" described in this section, not only the "lightweight" Expert Review that may apply to the 10000-64999 range.
+For this range, in addition to the checks described in {{checks}}, the DE is instructed to also evaluate the requested codepoint concerning the limited availability of the 1-byte codepoint space.
 For the 10000-64999 range, this criterion does not apply.
 
 ## "Lightweight" Expert Review Checks {#checks}
 
-The "lightweight" Designated Expert review checklist for the CoAP Content-Formats registry consists of the following steps:
+For the 10000-64999 range, the DE is instructed to perform the "lightweight" Expert review, as described by the following checklist:
```

Comment:
"DE" needs to be written in full here. This was the result of an earlier change I proposed, related to the typical usage of this text section that would get accessed by most readers from a direct link from the IANA registry. For that reason it should be readable without jumping back to find acronyms. 

```suggestion
For the 10000-64999 range, the Designated Expert is instructed to perform the "lightweight" Expert review, as described by the following checklist:
```